### DISCUSSION TITLE
Find Login & form UI template

### DISCUSSION THREAD

**Discussion by tobyyu007** [ResilientApp/Res-a-Me#5] [Ideas]
The template need to have
- Login and Register Page
- Form that can be used for the edit button

**Comment by tsungchiehchen** [comment]
- [Vue Simple Login](https://github.com/ameyrupji/vue-js-simple-login-app/tree/master)
    - Runnable
- [Mimic Google Login](https://github.com/lhsazevedo/vuetify-material-auth)
- [Another](https://github.com/omar-labana/instabug-task/tree/dev)
- https://primevue.org/inputmask/
    - Seems to have multiple components that can be used directly
    - Has Input box that can act as signup or login text input box
    - Has autocomplete search box

**Comment by yingchengchen** [comment]
- https://github.com/Jenesius/vue-form
    - simple form
- https://codepen.io/sethdavis512/pen/VPwygN
    - drop down/ slider/ selection

**Comment by tobyyu007** [comment]
# Nov 10 Discussion Result

## May use [Resume Builder's Layout](https://vuejsexamples.com/a-simple-and-strong-resume-buidler-built-with-vue-js/)
<img width="1913" alt="image" src="https://github.com/tobyyu007/Res-a-Me/assets/39978173/b165a1ed-d34f-4971-a24e-53d20223b8ea">
